src/plugins/intel_cpu/tests/unit/shape_inference_test/adaptive_max_pool_shape_inference_test.cpp

```diff
@@ -32,8 +32,8 @@ TEST_F(AdaptiveMaxPoolV8StaticShapeInferenceTest, default_ctor) {
     EXPECT_THAT(output_shapes, Each(StaticShape({1, 3, 10, 20})));
 
     // implementation depends on some output information of the op
-    op->set_output_type(0, element::i32, {1, 3, 10, 20});
-    op->set_output_type(1, element::i32, {1, 3, 10, 20});
+    op->set_output_type(0, element::i32, {-1, -1, -1, -1});
```

Comment:
I think the `unit_test::cpu_test_shape_infer` can be removed from test when default ctor is used. 

This defautl ctor without args is special case to check that shape_infer function do job correctly but is not real case in CPU plugin is more specific case for GPU plugin.

Reply:
fixed